### PR TITLE
Add Google Cloud support

### PR DESCRIPTION
**Pull request by michael-chi**
In related to  #39 

* GitHub Actions as the CI/CD platform.
* Ability to generate ML project from the cookiecutter template. Which allows the user to configure:
   - ML Project
   - CI/CD platform (Github Actions)
   - Choose Google Cloud as the cloud provider
   - Enable Feature Store
   - Production / Staging workspaces
   - ML experiment parent directory
   - Github branches
* Integration between GitHub Actions and Google Cloud.
   - Authentication through Workload Identity Pool.
   - Setup and configure required Google Cloud permissions / roles through Terraform.
* Add required configuration and setup instructions to the readme.

### CODE REVIEW

File: {{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/provider.tf [hunk 1]
```diff
@@ -48,7 +48,16 @@ terraform {
     dynamodb_table = "{{cookiecutter.project_name}}-cicd-setup-tfstate-lock"
     key            = "cicd-setup.terraform.tfstate"
   }
-{%- endif %}
+{% elif cookiecutter.cloud == "gcp" -%}
+  // The `backend` block below configures the s3 backend
+  // (docs: https://www.terraform.io/language/settings/backends/s3)
+  // for storing Terraform state in an AWS S3 bucket. You can run the setup scripts in mlops-setup-scripts/terraform to
+  // provision the S3 bucket referenced below and store appropriate credentials for accessing the bucket from CI/CD.
```
Comment (stawo) [inline {{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/provider.tf:55]: Fix comment to properly address Google Cloud Storage

**Review by stawo — CHANGES_REQUESTED** [review]
Just few notes I managed to catch. Did not have time to completely review.

**Comment by arpitjasa-db** [comment]
Hi @michael-chi @stawo, we've made some pretty significant changes to MLOps Stacks since this PR was opened, would you be interested in updating your PR?

**Comment by stawo** [comment]
@arpitjasa-db , I don't own this PR, I just did a small and incomplete review of it 😄 
Let's see if @michael-chi will update the PR following the latest changes.

**Comment by arpitjasa-db** [comment]
Closing this since we now support GCP with https://github.com/databricks/mlops-stacks/pull/153

**Pull request closed without merging**